freqtrade/freqtradebot.py

```diff
@@ -67,8 +67,15 @@ def __init__(self, config: Dict[str, Any]) -> None:
 
         # Check config consistency here since strategies can set certain options
         validate_config_consistency(config)
-
-        self.exchange = ExchangeResolver.load_exchange(self.config['exchange']['name'], self.config)
+        
+        unwanted_exchanges = ['coinex']
+        exchange_name = self.config['exchange']['name']
+        
+        if exchange_name in unwanted_exchanges:
+          logger.error(f'Exchange, {exchange_name} not supported by freqtrade! continue with your own risk!')
```

Comment:
If an exchange is REALLY not working (confirmed not working) - then please add it to the `BAD_EXCHANGES` Dict in `freqtrade/exchange/common.py` with the reason for it - which will have the same effect.